themes/harper/layouts/partials/footer.html

```
@@ -0,0 +1,6 @@
+<footer class="footer">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)